### PR TITLE
[MINI-4355] contact fields name / email fix

### PR DESCRIPTION
**Pull request by laubengaier**
# Description
Small bugfix required by QA to change responses depending on specified error cases

## Links
https://jira.rakuten-it.com/jira/browse/MINI-4355

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [ ] I ran `fastlane ci` without errors

### CODE REVIEW

File: .swiftlint.yml [hunk 1]
```diff
@@ -8,8 +8,8 @@ line_length:
     ignores_comments: true
     ignores_urls: true
 function_body_length:
-    warning: 40
-    error: 60
+    warning: 60
+    error: 100
```
Comment (Climbatize) [inline .swiftlint.yml:12]: I'd prefer you disable it locally

**Review by Climbatize — CHANGES_REQUESTED** [review]

File: .swiftlint.yml (hunk 1 above)
Comment (laubengaier) [reply .swiftlint.yml:12]: will the ci not check for the same?

Comment (Climbatize) [reply .swiftlint.yml:12]: Oh sorry I'm not clear enough, disable lint body length check on your method:
```// swiftlint:disable function_body_length```

Comment (laubengaier) [reply .swiftlint.yml]: fixed it thanks for the reminder!

**Review by Climbatize — APPROVED** [review]

**Review by rleojoseph — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**